homeassistant/components/androidtv/media_player.py

```diff
@@ -586,6 +566,11 @@ class AndroidTVDevice(ADBDevice):
 
     _attr_supported_features = SUPPORT_ANDROIDTV
 
+    @property
+    def device_info(self):
+        """Return a device description for device registry."""
+        return self._get_device_info("AndroidTV")
+
     @adb_decorator(override_available=True)
     async def async_update(self):
         """Update the device state and, if necessary, re-connect."""
```

Comment:
```suggestion
    @adb_decorator(override_available=True)
    async def async_update(self):
        """Update the device state and, if necessary, re-connect."""
        self._attr_device_info = self._get_device_info("AndroidTV")

```

Reply:
I think that `device_info` should be called before than first update, in this way was skipping some rows in tests. I restored the the property.

Reply:
What if you add `,True` to `async_add_entities`? Does it work then?

Reply:
Think is better to not change this, but I found a different solution to use attribute.

Reply:
Seems that this attribute must be provided before added to hass or device is nor created. Now should be ok,